Certified SAIL Extension (CSE) Examples/mapField/mapField/v2/mapField_en_US.properties

```diff
@@ -0,0 +1,6 @@
+name=[v2]Map
```

Comment:
Same comment as above, the v2 example here seems like it may be more confusing for an example reference CSE

Reply:
@john-irby I'm just using our latest known good example from our test harness. I do think it's valuable to have an example which demonstrates CSE evolution. Perhaps it's worthwhile to split out into a separate example and keep mapField as the basic example?

Reply:
I think making two separate examples with mapField as a base would be best - we could also update version 2.0.0 of the mapField CSE to have noticeably different behavior which would better illustrate the evolution example (though that should probably be split out from this PR)

Reply:
Removed 2.0.0 in favor of maintaining a bare bones example to get people started with the concepts of CSEs.